packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -10,12 +10,12 @@ import { Semver } from "../universal/Semver.sol";
  * @title L1StandardBridge
  * @notice The L1StandardBridge is responsible for transfering ETH and ERC20 tokens between L1 and
  *         L2. In the case that an ERC20 token is native to L1, it will be escrowed within this
- *         contract. If the ERC20 token is native to L2, it will be burnt.
- *         ETH is transferred to and escrowed within the OptimismPortal contract.
- *         Note that this contract is not intended to support all variations of ERC20 tokens.
- *         Examples of some token types that may not be properly supported by this contract include,
- *         but are not limited to: tokens with transfer fees, rebasing tokens, and
- *         tokens with blocklists.
+ *         contract. If the ERC20 token is native to L2, it will be burnt. After the Bedrock
```

Comment:
Perhaps frame this as "before bedrock, ETH was stored within this contract. After bedrock, ETH is stored inside of the OptimismPortal"

This gives better context